packages/firebase_auth/firebase_auth_platform_interface/lib/src/firebase_auth_exception.dart

```diff
@@ -11,30 +11,24 @@ import 'auth_credential.dart';
 class FirebaseAuthException extends FirebaseException implements Exception {
   // ignore: public_member_api_docs
   @protected
-  FirebaseAuthException(
-      {@required this.message,
-      @required this.code,
-      this.email,
-      this.credential,
-      this.phoneNumber,
-      this.tenantId})
-      : super(plugin: 'firebase_auth', message: message, code: code);
-
-  /// Unique error code
-  final String code;
-
-  /// Complete error message.
-  final String message;
+  FirebaseAuthException({
+    String? message,
```

Comment:
"message" is optional/nullable in base-classes, and some places passed `null` as message, so I removed the `required`

Reply:
`required` removal makes sense 👍